client/src/components/ClubMap.vue

```diff
@@ -21,6 +21,11 @@ const props = defineProps({
   },
 });
 
+// Find a way to make this reactive
```

Comment:
Kannst du das nicht in ein simples Composable mit dieser einen Funktion auslagern?